base/services/w32time/w32time.c

```diff
@@ -153,7 +153,7 @@ GetIntervalSetting(VOID)
     }
 
     if (lRet != ERROR_SUCCESS || dwData < 120)
-        return 9 * 60 * 60; // 9 hours, because Windows uses 9 hrs, 6 mins and 8 seconds by default.
+        return W32TIME_POLL_INTERVAL;
```

Comment:
```suggestion
        return W32TIME_POLL_INTERVAL; // HACK value (CORE-19281), instead of 9 h 6 min 8 s Windows default.
```

Reply:
Is Windows using different defaults here (lol, why "9h 6mins 8secs" and not 9 seconds or 6h 6mins 6secs" or any other random value??! ^^) than elsewhere (the 7 days in the other places) ?

Reply:
I do not believe that this is a HACK. Instead of a HACK, I suggest that is is a PREFERENCE. I do not know the actual reason that Microsoft chose to use this value, but I have a hunch. I believe that the responsible developer had a child with a birthday of 6-Sep-2008 (US 09/06/08) and chose this value to immortalize it in code. :).
Does anyone have a reason why the 4 hours that I chose is any kind of problem?

Reply:
I still see no real reason why we should use a different interval than MS Windows. Faster interval means additional OS load, (and is most likely only chosen to hide bugs somewhere else). I am no real fan of this PR.

Reply:
> Faster interval means additional OS load

It's not as if it was a time update every minute or so.

Reply:
`Is it possible to ensure that the time service never tries to set the time back into the past?`
I think that would require to make our clock by default run a bit slower than walltime-clock. It's not that easy, as the clock generators on a mainboard do have hardware-fluctuations (monday-samples) and the speed of those clocks also depends on the temperature with the PC-tower. I don't think such a ahead-mitigation is possible.

But yes: in case we would ever have kind of a rounding-decision to take "either making the clock running too fast or making it run too slow", then making it running too slow would be the better default choice.

Reply:
@JoachimHenze, I see your concern for trying to set the time back into the past, but from my experience, it is not really a problem. This does happen on a normal basis whenever DST time is in effect and the time changes back from DST to Standard Time (CST - Central Standard Time for where I live). In my case when this happens, the time goes from 2:00 AM back to 1:00 AM. At one time this could cause computer problems for logging, but has been accounted for now in most operating systems.
The MAIN reason that I wanted to make this change is that on DST transitions, it can be many days before the clock is updated to the correct time the way that ReactOS works now. So maybe this is just a temporary workaround until we can get notifications or another way of updating the clock on these transitions. Thanks.